UIKitDZ/UIKitDZ/ViewController.swift

```diff
@@ -5,9 +5,188 @@ import UIKit
 
 /// ViewController
 class ViewController: UIViewController {
-//    override func viewDidLoad() {
-//        super.viewDidLoad()
+    lazy var guessButton: UIButton = {
```

Comment:
```suggestion
    private lazy var guessButton: UIButton = {
```
Нужно добавить комментарии на все свойства и методы, как на скрине.
<img width="646" alt="Снимок экрана 2024-02-11 в 14 08 41" src="https://github.com/KadyrKZ/UIKitDZ/assets/149266332/7b359db1-9d02-4933-ab2b-ab50c1cd70de">